--- origsrc/wxMaxima-0.7.5/configure.in	2008-04-19 09:29:52.000000000 -0500
+++ src/wxMaxima-0.7.5/configure.in	2008-08-22 00:55:17.631253500 -0500
@@ -24,9 +24,6 @@
         *mingw*)
                 win32=true
                 ;;
-        *cygwin*)
-                win32=true
-                ;;
         *)
                 win32=false
 esac
--- origsrc/wxMaxima-0.7.5/src/main.cpp	2008-01-28 06:40:11.000000000 -0600
+++ src/wxMaxima-0.7.5/src/main.cpp	2008-08-22 01:26:09.339437000 -0500
@@ -29,12 +29,14 @@
 // On wxGTK2 we support printing only if wxWidgets is compiled with gnome_print.
 // We have to force gnome_print support to be linked in static builds of wxMaxima.
 
+#ifndef __CYGWIN__
 #if defined wxUSE_LIBGNOMEPRINT
  #if wxUSE_LIBGNOMEPRINT
   #include "wx/html/forcelnk.h"
   FORCE_LINK(gnome_print)
  #endif
 #endif
+#endif
 
 
 IMPLEMENT_APP(MyApp)
